perfkitbenchmarker/scripts/database_scripts/launch_mysql_service.py

```diff
@@ -47,12 +51,24 @@
 import time
 import gflags
 
+import file_to_plot
+
 # GLOBAL STRINGS
+PER_SECOND_GRAPHS = 'per_second_graphs'
```

Comment:
No this is the string constant for the flag name. This should be a string. Agreed?